osm2pgsql.cpp

```diff
@@ -45,6 +46,7 @@
 
 int main(int argc, char *argv[])
 {
+    std::setlocale(LC_NUMERIC, "C");
```

Comment:
I'm not sure what other side effects a global change of the locale has. If it is running fine on Windows without this, I'd say we just go with the _configthreadlocale below.